src_analysis/fuzzer_dependencies.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module for finding dependencies of fuzzers, and fuzzers that are
+dependent on given files."""
+import importlib
+import inspect
+import os
+import types
+from typing import List
+import sys
+
+from common import fuzzer_utils
+
+MAX_DEPTH = 5
+PY_DEPS_CACHE = {}
+
+
+def _get_fuzzer_module(fuzzer: str) -> str:
+    """Returns the module fuzzer.py module of |fuzzer|."""
+    return 'fuzzers.{}.fuzzer'.format(fuzzer)
+
+
+def is_builtin_module(module) -> bool:
+    """Returns the module for |fuzzer|'s fuzzer.py."""
+    return module.__name__ in sys.builtin_module_names
```

Comment:
Maybe this is too fragile since it will incorrectly return true if I have a module blah.time?